packages/sakuli-legacy/src/context/sahi/accessor/accessor-util.class.ts

```diff
@@ -253,11 +253,17 @@ export class AccessorUtil {
     return identifier.startsWith("/") && identifier.endsWith("/");
   }
 
+  /**
+   * Allows the user to use an index at the end of the string, e.g.
```

Comment:
Maybe we should add a note mentioning that the index is extracted via the capturing group `([0-9]+)`